docs/source/executing-operations/query-batching.mdx

```diff
@@ -1,34 +1,45 @@
 ---
 title: Query batching
-description: Receive query batches with the Apollo Router  
+description: Receive query batches with the Apollo Router
 ---
 
-<ExperimentalFeature />
```

Comment:
We need to add the Enterprise tag to this feature

Reply:
done: feb1769b41c3f652d27701ff94b08fcd0da19f3d